spec/logged_release_spec.rb

```diff
@@ -55,11 +66,18 @@ module Fauxpaas
       )
     end
 
+    before(:each) do
+      Fauxpaas.config.tap do |canister|
+        canister.register(:release_time_format) { Fauxpaas.settings.release_time_format }
+      end
+    end
```

Comment:
It doesn't seem like this should be necessary. It's getting pulled in from settings at startup (fauxpaas.rb), right? If you wanted to be extra-sure that a particular value is used, `config/settings/test.yml` would be the way to go, but I still wouldn't think that should require a block here.

Reply:
No, see my other comment.